service/api_task.go

```diff
@@ -485,7 +485,12 @@ func assignNextAvailableTask(ctx context.Context, taskQueue *model.TaskQueue, di
 			return nil, false, errors.Wrapf(err, "could not find project ref for next task %s", nextTask.Id)
 		}
 
-		if !projectRef.IsEnabled() || projectRef.IsDispatchingDisabled() {
+		isDisabled := projectRef.IsDispatchingDisabled()
+		if !projectRef.IsEnabled() && (queueItem.Requester != evergreen.GithubPRRequester || !projectRef.IsHidden()) {
```

Comment:
should the hidden check be inverted? I would think that a hidden project shouldn't run normal tasks

Reply:
Hidden projects can run PR tasks, so this means: It's disabled if the requester ISN'T for PRs, or if the requester is for PR but the project isn't hidden (since PRs are the special case).

Reply:
I realize this is a confusing check to think through but I couldn't figure out how to write it better. I'll add a comment.